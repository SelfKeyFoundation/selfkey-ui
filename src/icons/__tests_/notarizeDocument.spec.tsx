import * as React from 'react';
import { create } from 'react-test-renderer';
import NotarizeDocumentIcon from '../notarize-document';

describe('Notarize Document Icon', () => {
  it('should show Notarize Document Icon', () => {
    const component = create(
      <NotarizeDocumentIcon />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});