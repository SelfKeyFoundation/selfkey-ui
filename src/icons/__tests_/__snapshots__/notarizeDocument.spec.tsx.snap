// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notarize Document Icon should show Notarize Document Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root-1 Component-root-0-1-1"
  focusable="false"
  role="presentation"
  style={
    Object {
      "height": "18px",
      "width": "16px",
    }
  }
  viewBox="0 0 16 18"
>
  <g
    fillRule="evenodd"
    id="Notarize-document-icon"
    stroke="none"
    stroke-width="1"
  >
    <g
      fill="#FFFFFF"
      id="nt-d-i"
      transform="translate(-1011.000000, -396.000000)"
    >
      <g
        id="nt-d-i-m"
        transform="translate(150.000000, 264.000000)"
      >
        <g
          id="nt-d-i-mb"
          transform="translate(0.000000, 71.000000)"
        >
          <g
            id="nt-d-i-mbr"
            transform="translate(29.000000, 48.000000)"
          >
            <g
              id="Button/Full-Primary"
              transform="translate(812.000000, 0.000000)"
            >
              <g
                id="text"
                transform="translate(20.000000, 12.000000)"
              >
                <path
                  d="M14.7368423,1 C15.3908636,1 15.9287922,1.49705223 15.9934785,2.13400719 L16,2.26315773 L16,17.4210505 C16,18.0750718 15.5029478,18.6130004 14.8659928,18.6776867 L14.7368423,18.6842082 L1.26315981,18.6842082 C0.609138479,18.6842082 0.0712099175,18.187156 0.00652361819,17.550201 L2.07722234e-06,17.4210505 L2.07722234e-06,6.05263092 C-0.000713968081,5.65696352 0.18373113,5.28692265 0.493020891,5.04909498 L0.61473884,4.96631527 L3.98315945,1.59789466 C4.18715699,1.26862526 4.52920694,1.05348676 4.90874527,1.00869999 L5.052633,1 L14.7368423,1 Z M14.7200002,2.27999983 L5.07789615,2.27999983 L5.07789615,2.30526299 L5.052633,2.33052614 L5.052633,6.06947303 L1.31368612,6.06947303 L1.28842296,6.09473618 L1.26315981,6.09473618 L1.26315981,17.4210505 L14.7368423,17.4210505 L14.7200002,2.27999983 Z M3.78947527,13.6315773 L10.9473691,13.6315773 C11.2961805,13.6315773 11.5789479,13.9143448 11.5789479,14.2631562 C11.5789479,14.5802574 11.3452558,14.842777 11.0406992,14.8878871 L10.9473691,14.894735 L3.78947527,14.894735 C3.44066389,14.894735 3.1578964,14.6119675 3.1578964,14.2631562 C3.1578964,13.9460549 3.39158854,13.6835354 3.69614517,13.6384252 L3.78947527,13.6315773 L10.9473691,13.6315773 Z M12.2105268,10.6842093 C12.5593382,10.6842093 12.8421057,10.9669768 12.8421057,11.3157881 C12.8421057,11.6328894 12.6084135,11.8954089 12.3038569,11.9405191 L12.2105268,11.947367 L3.78947527,11.947367 C3.44066389,11.947367 3.1578964,11.6645995 3.1578964,11.3157881 C3.1578964,10.9986869 3.39158854,10.7361674 3.69614517,10.6910572 L3.78947527,10.6842093 L12.2105268,10.6842093 Z M11.7894742,3.94736804 C12.487097,3.94736804 13.052632,4.51290302 13.052632,5.21052577 L13.052632,5.21052577 L13.052632,7.73684123 C13.052632,8.43446398 12.487097,8.99999896 11.7894742,8.99999896 L11.7894742,8.99999896 L9.26315877,8.99999896 C8.56553602,8.99999896 8.00000104,8.43446398 8.00000104,7.73684123 L8.00000104,7.73684123 L8.00000104,5.21052577 C8.00000104,4.51290302 8.56553602,3.94736804 9.26315877,3.94736804 L9.26315877,3.94736804 Z M11.7894742,5.21052577 L9.26315877,5.21052577 L9.26315877,7.73684123 L11.7894742,7.73684123 L11.7894742,5.21052577 Z M3.78947527,3.57684177 L2.57684385,4.78947319 L3.78947527,4.78947319 L3.78947527,3.57684177 Z"
                  id="nt-d-i-icon-document"
                />
              </g>
            </g>
          </g>
        </g>
      </g>
    </g>
  </g>
</svg>
`;
